packages/sonarwhal/src/lib/utils/resource-loader.ts

```diff
@@ -244,11 +245,13 @@ const generateConfigPathsToResources = (configurations: Array<string>, name: str
  */
 export const loadResource = (name: string, type: ResourceType, configurations: Array<string> = [], verifyVersion = false) => {
     debug(`Searching ${name}…`);
-
+    const isSource = fs.existsSync(name); // eslint-disable-line no-sync
```

Comment:
if name is a relative path, this is not going to work, isn't it?

Reply:
It was working on my local tests. IIRC `fs. exists` uses `process.cwd()` so it should be ok?
I guess the problem is if we are calling from another place different place than where the `.sonarwhalrc` file is located (e.g.: when using the file in the user's home folder)